diff --git a/node_modules/hardhat-deploy/dist/src/helpers.js b/node_modules/hardhat-deploy/dist/src/helpers.js
index d767d62..1d50273 100644
--- a/node_modules/hardhat-deploy/dist/src/helpers.js
+++ b/node_modules/hardhat-deploy/dist/src/helpers.js
@@ -597,7 +597,7 @@ function addHelpers(deploymentManager, partialExtension, network, getArtifact, s
         }
         const proxyName = name + '_Proxy';
         const { address: owner } = getProxyOwner(options);
-        const { address: from } = getFrom(options.from);
+        // const { address: from } = getFrom(options.from);
         const implementationArgs = options.args ? [...options.args] : [];
         // --- Implementation Deployment ---
         const implementationName = name + '_Implementation';
